### DISCUSSION TITLE
MD5 ("checksum") not found in document index (fscrawler 2.8) - documentation update

### DISCUSSION THREAD

**Discussion by cbb-colab** [dadoonet/fscrawler#1343] [Q&A]
Running both fscrawler 2.8 (FS) and elasticsearch 7.16.2 (ES7) in docker container

Just want to index file meta data INCLUDING a MD5 checksum for each file, no content, neither TIKA metadata needed.

After several try-and-errors and looking in the source I was able to get the checksum field generated in index and filled by using

```yaml
fs:
  index_content: true
  indexed_chars: 0
  checksum: "MD5"
```

in `_settings.yaml`, where the magic (for me) was to set `index_content` to `true` where I do not want to index content at all.
By setting `indexed_chars` to `0` I avoid filling up my index.

My recommendation would be to document "checksum" will only work if "index_content" is set to true at all.

**Comment by dadoonet** [comment]
Very nice. Thanks for sharing ;)

Would you like to open a doc PR for this? I think this could go in https://fscrawler.readthedocs.io/en/fscrawler-2.8/user/tips.html. WDYT?

Or indeed in the checksum part of the documentation.

**Reply by cbb-colab** [reply]
I will do a PR and try to add a comment into the checksum option of the documentation, as well as to add the "tip" with zero-size content into the tip section

**Reply by cbb-colab** [reply]
I now made a fork with the changes I propose (index.rst, local-fs.rst) (https://github.com/cbb-colab/fscrawler/tree/origin/improve-documentaion) and hope you are able to merge the twio files (and cross check)
